org.eclipse.xtext.builder.standalone/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -1,53 +1,60 @@
-#Wed Oct 14 22:36:24 CEST 2009
 cleanup.add_default_serial_version_id=true
 cleanup.add_generated_serial_version_id=false
 cleanup.add_missing_annotations=true
 cleanup.add_missing_deprecated_annotations=true
 cleanup.add_missing_methods=false
 cleanup.add_missing_nls_tags=false
 cleanup.add_missing_override_annotations=true
+cleanup.add_missing_override_annotations_interface_methods=true
 cleanup.add_serial_version_id=true
 cleanup.always_use_blocks=true
 cleanup.always_use_parentheses_in_expressions=false
 cleanup.always_use_this_for_non_static_field_access=false
 cleanup.always_use_this_for_non_static_method_access=false
-cleanup.convert_to_enhanced_for_loop=false
-cleanup.correct_indentation=false
-cleanup.format_source_code=false
+cleanup.convert_functional_interfaces=true
+cleanup.convert_to_enhanced_for_loop=true
+cleanup.correct_indentation=true
+cleanup.format_source_code=true
 cleanup.format_source_code_changes_only=false
-cleanup.make_local_variable_final=true
+cleanup.insert_inferred_type_arguments=false
+cleanup.make_local_variable_final=false
 cleanup.make_parameters_final=false
 cleanup.make_private_fields_final=true
 cleanup.make_type_abstract_if_missing_method=false
-cleanup.make_variable_declarations_final=false
+cleanup.make_variable_declarations_final=true
 cleanup.never_use_blocks=false
 cleanup.never_use_parentheses_in_expressions=true
-cleanup.organize_imports=false
+cleanup.organize_imports=true
 cleanup.qualify_static_field_accesses_with_declaring_class=false
 cleanup.qualify_static_member_accesses_through_instances_with_declaring_class=true
 cleanup.qualify_static_member_accesses_through_subtypes_with_declaring_class=true
 cleanup.qualify_static_member_accesses_with_declaring_class=true
 cleanup.qualify_static_method_accesses_with_declaring_class=false
 cleanup.remove_private_constructors=true
+cleanup.remove_redundant_modifiers=true
+cleanup.remove_redundant_semicolons=true
+cleanup.remove_redundant_type_arguments=true
 cleanup.remove_trailing_whitespaces=true
-cleanup.remove_trailing_whitespaces_all=true
-cleanup.remove_trailing_whitespaces_ignore_empty=false
+cleanup.remove_trailing_whitespaces_all=false
+cleanup.remove_trailing_whitespaces_ignore_empty=true
 cleanup.remove_unnecessary_casts=true
 cleanup.remove_unnecessary_nls_tags=true
-cleanup.remove_unused_imports=false
-cleanup.remove_unused_local_variables=false
+cleanup.remove_unused_imports=true
+cleanup.remove_unused_local_variables=true
```

Comment:
why is this changed?

Reply:
These are the cleanup options I have used to clean up the Java sources. I'll remove the file change here.